### PR TITLE
final2: InvalidCommandTest: fix invalidAntNameTest's escape test

### PR DESCRIPTION
**Pull request by pcworld**
Use task sheet field for invalidAntNameTest's create test
Fix typos

### CODE REVIEW

File: src/final2/subtests/InvalidCommandTest.java [hunk 1]
```diff
@@ -155,17 +155,11 @@ public void createExistingAntTest() {
...
-					"0000",
-					"0a00",
-					"0000"
-			};
 			runs = new Run[] {
 					new ErrorRun("create " + invalidAntName + ",1,1"),
 					quit()
 			};
-			sessionTest(runs, Input.getFile(inputFile));
+			sessionTest(runs, Input.getFile(TASK_SHEET_INPUT_FILE_1));
```
Comment (RomanLangrehr) [inline src/final2/subtests/InvalidCommandTest.java:162]: I do not agree on using TASK_SHEET_INPUT_FILE_1 here:
TASK_SHEET_INPUT_FILE_1 should only be used, when the playing field is not important. Here it is important that the cell 1,1 is empty and not an obstacle. (In the other examples the palying field is not important at all...)

Comment (pcworld) [inline src/final2/subtests/InvalidCommandTest.java:162]: > TASK_SHEET_INPUT_FILE_1 should only be used, when the playing field is not important

Why?
In TASK_SHEET_INPUT_FILE_1, cell `1,1` is "empty" and not an obstacle.

Comment (RomanLangrehr) [inline src/final2/subtests/InvalidCommandTest.java:162]: > In TASK_SHEET_INPUT_FILE_1, cell 1,1 is "empty" and not an obstacle

But I don't see this (immidiatly) in the test. I think it's best, when you can understand every test without knowing anything about TASK_SHEET_INPUT_FILE_1 (except that it's an valid playing field). (Due to clarity)

**Comment by jGleitz** [comment]
Merging in favour of #172. Please don't stop the discussion.

File: src/final2/subtests/InvalidCommandTest.java (hunk 1 above)
Comment (pcworld) [inline src/final2/subtests/InvalidCommandTest.java:162]: So revert this? And what about TASK_SHEET_INPUT_FILE_1 in the `escape` test?

Comment (RomanLangrehr) [inline src/final2/subtests/InvalidCommandTest.java:162]: I'd rather revert this. But in the escape test, any well-formed playing field is okay, so I'd keep TASK_SHEET_INPUT_FILE_1 there.

Comment (pcworld) [inline src/final2/subtests/InvalidCommandTest.java:162]: Pushed revert to master: 5be1769af127700dbfc882c0c08269af14c235e6

Comment (RomanLangrehr) [inline src/final2/subtests/InvalidCommandTest.java:162]: Allright, thanks.
By the way: It was a really good job finding these bugs here.

**Pull request merged**